de/translation.json

```diff
@@ -102,10 +102,10 @@
   "Activity.State.HydratingReportsCache": "Berichtsdaten werden aus dem Gerätespeicher geladen.",
   "Activity.State.LoadingPeriod": "Scanne... {{date}}",
   "Activity.State.NoBookmarks": "Du hast noch keine Lesezeichen hinzugefügt!\n\n Du kannst deine Lesezeichen synchronisieren, indem du die Synchronisierung in den _Einstellungen_ aktivierst.",
-  "Activity.State.NoHistoryAvailable": "🦘 Find a fireteam and seek triumph against enemies of the Traveler.",
+  "Activity.State.NoHistoryAvailable": "Finde einen Einsatztrupp und erringe einen Triumph gegen die Feinde des Reisenden.",
   "Activity.State.PeriodNotFound": "Der angeforderte Zeitraum konnte in der Aktivitätshistorie dieses Charakters nicht gefunden werden.",
   "Activity.State.RequestHistory": "Analyse des Aktivitätsverlaufs",
-  "Activity.State.RequestHistoryPage": "🦘 Retrieving activity history volume {{page}}",
+  "Activity.State.RequestHistoryPage": "Aktivitätsverlauf wird abgerufen: Seite {{page}}",
```

Comment:
While creative, this phrasing might indicate a state instead of an ongoing process. A suggestion would be:

> Rufe Seite ${{page}} des Aktivitätsverlaufs ab

or if you want to keep it at the end, maybe

> Aktivitätsverlauf wird abgerufen – lade Seite {{page}}